apps/web/src/components/GalleryEditor/CollectionEditor/StagedNftImageDragging.tsx

```diff
@@ -77,80 +59,30 @@ function StagedNftImageDraggingWithoutFallback({
     tokenRef
   );
 
-  const imageUrl = useGetSinglePreviewImage({ tokenRef: token, size: 'large' });
+  const imageUrl = useGetSinglePreviewImage({ tokenRef: token, size: 'large' }) ?? '';
```

Comment:
this is an unusual new pattern that's introduced- even though we know the `imageUrl` will either be a string or return an error, typescript doesn't know this because this is conditional based on the `shouldThrow` being `true` - which is the default case

made a ticket for this and will come back to it